okhttp/src/main/java/com/squareup/okhttp/internal/spdy/SpdyConnection.java

```diff
@@ -640,8 +640,11 @@ private Reader() {
             @Override public void execute() {
               try {
                 handler.receive(newStream);
-              } catch (IOException e) {
-                throw new RuntimeException(e);
+              } catch (RuntimeException | IOException e) {
```

Comment:
So now we're swallowing the exceptions thrown by the higher-level handler. I don't think this is what we want to do here; we're hiding bugs.